test/import.js

```diff
@@ -44,17 +46,42 @@ exports.import = {
         test.done();
     },
 
-    "supports a Date instance": function(test) {
+    "supports a Date instance greater than 5 days old": function(test) {
         var event = "test",
-            time = new Date,
+            six_days_ago = mock_now_time - 1000 * 60 * 60 * 24 * 6,
+            time = new Date(six_days_ago),
             props = { key1: 'val1' },
             expected_endpoint = "/import",
             expected_data = {
                 event: 'test',
                 properties: {
                     key1: 'val1',
                     token: 'token',
-                    time: 0
+                    time: six_days_ago / 1000
+                }
+            };
+
+        this.mixpanel.import(event, time, props);
+
+        test.ok(
+            this.mixpanel.send_request.calledWithMatch(expected_endpoint, expected_data),
+            "import didn't call send_request with correct arguments"
+        );
+
+        test.done();
+    },
+
+    "supports a Date instance less than 5 days old": function(test) {
+        var event = "test",
+            time = new Date(mock_now_time),
+            props = { key1: 'val1' },
+            expected_endpoint = "/track",
```

Comment:
Hmm, I would say that if the lib user is calling `mixpanel.import()`, it should always go to the `import` endpoint, regardless of time. I would probably do the same for `mixpanel.track()` (always go to `/track`, and throw an exception if it's out of range). This would require a bit of refactoring, because right now `mixpanel.import()` ends up calling `mixpanel.track()`, rather than both relying on shared code before calling `send_request()`. Does that seem doable?